pilot/pkg/proxy/envoy/v1/watcher.go

```diff
@@ -55,27 +55,30 @@ type CertSource struct {
 }
 
 type watcher struct {
-	agent    proxy.Agent
-	role     model.Node
-	config   meshconfig.ProxyConfig
-	certs    []CertSource
-	pilotSAN []string
+	agent         proxy.Agent
+	role          model.Node
+	config        meshconfig.ProxyConfig
+	requiredCerts []CertSource // These certs must present for Envoy to start
+	certs         []CertSource
+	pilotSAN      []string
 }
 
 // NewWatcher creates a new watcher instance from a proxy agent and a set of monitored certificate paths
 // (directories with files in them)
 func NewWatcher(config meshconfig.ProxyConfig, agent proxy.Agent, role model.Node,
-	certs []CertSource, pilotSAN []string) Watcher {
+	internalCerts []CertSource, externalCerts []CertSource, pilotSAN []string) Watcher {
```

Comment:
`internalCerts` and `externalCerts` have the same type and can be grouped, i.e.
```golang
..., role model.Node, internalCerts, externalCerts []CertSource, pilotSAN []string)
```

Reply:
Done.